### DISCUSSION TITLE
A simple way to access a small classic dataset from datatable

### DISCUSSION THREAD

**Discussion by oleksiyskononenko** [h2oai/datatable#2676] [Ideas]
What if... we bring several small classic datasets to datatable, so that we could use them in our tutorials/tests by simply doing something like `from datatable import DT_iris, DT_credit_card`.

**Comment by st-pasha** [comment]
I would prefer not to pollute the global namespace with too many names, so how about we have a function that loads the named dataset, like
```
dt.example('iris')
dt.example('creditcard')
```

**Reply by oleksiyskononenko** [reply]
Yes, the access could be whatever we think is relevant... I've been also thinking about simplifying access to the classic datasets in general through smth like `dt.load('airlines')`, but it seems that there are no standard names and URLs where these datasets could be found.

**Reply by st-pasha** [reply]
We need a function name that makes it clear that it's for sample datasets only, not for loading generic data. So `dt.load()` is not good. On the other hand, i don't quite like `dt.example()` either...

**Reply by oleksiyskononenko** [reply]
`dt.load()` was a different thing, it was another idea of having a simple way to load a classic dataset from the web... but when I checked, there are no standard names for these datasets, so this idea makes no sense.

**Comment by st-pasha** [comment]
There is a public repository of various datasets: https://archive.ics.uci.edu/ml/datasets.php 
There are lots of examples there, so we might as well choose something that has not yet bored everyone to death :)

**Reply by oleksiyskononenko** [reply]
Yeah, we definitely have a choice as for the small datasets for tutorials :) I would even use one in the FTRL tests... but if I want to have a shortcut to load airlines or any other dataset, like through a possible `dt.load()` I have mentioned above, it could become kind of messy. It is just simpler to load the data by URL.

**Comment by st-pasha** [comment]
For example this one looks good: https://archive.ics.uci.edu/ml/machine-learning-databases/flags/flag.data
